WordPress/Classes/Extensions/Font/UIConfigurationTextAttributesTransformer+Font.swift

```diff
@@ -0,0 +1,13 @@
+
+extension UIConfigurationTextAttributesTransformer {
+
+    /// Instance method that sets the font on the ``AttributeContainer``
+    static func transformer(with font: UIFont) -> UIConfigurationTextAttributesTransformer {
+        return UIConfigurationTextAttributesTransformer { incoming in
+            var outgoing = incoming
+            outgoing.font = font
+            return outgoing
+        }
+    }
```

Comment:
There are a few other places that use the same code, which is why I made this extension. I didn't update those locations though since this is going to a release branch and don't want to introduce more risk than necessary.